src/quacc/calculators/espresso/espresso.py

```diff
@@ -95,6 +100,34 @@ def write_input(
             **parameters,
         )
 
+    def _test_run(self, parameters: dict[str, Any], directory: Path):
```

Comment:
Add in `-> dict[str, Any]` for the return. Also, please turn the comment into a docstring even though it's a private function :)